.circleci/config.yml

```diff
@@ -810,6 +810,15 @@ jobs:
       - *install_otp
       - *install_libsodium
       - *restore_machine_build_cache
+      # Git based deps are problematic for rebar3 - even running "./rebar3 as system_test upgrade" doesn't do the job of ensuring these deps are up to date
```

Comment:
No this is not a general solution, it will work only in your case (updating sophia and it's CLI).

Please bump the machine cache version here: https://github.com/aeternity/aeternity/blob/master/.circleci/config.yml#L126-L133